pkg/metallb/metallb_test.go

```diff
@@ -366,6 +366,37 @@ func TestMetalLbWithOptions(t *testing.T) {
 	assert.Equal(t, "error", testBuilder.errorMsg)
 }
 
+func TestMetalLbWithFRRconfigAlwaysBlock(t *testing.T) {
+	testCases := []struct {
+		testMetalLb   *Builder
+		prefixes      string
+		expectedError string
+	}{
+		{
+			testMetalLb:   buildValidMetalLbBuilder(buildMetalLbTestClientWithDummyObject()),
+			expectedError: "",
+			prefixes:      "192.168.1.0/24",
+		},
+		{
+			testMetalLb:   buildValidMetalLbBuilder(buildMetalLbTestClientWithDummyObject()),
+			expectedError: "",
```

Comment:
No error message? not sure what're we testing here

Reply:
Added error message